examples/platform/esp32/time/TimeSync.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "TimeSync.h"
+#include <esp_sntp.h>
+#include <lib/support/CHIPMemString.h>
+#include <lib/support/logging/CHIPLogging.h>
+
+static constexpr time_t kMinValidTimeStampEpoch = 1704067200; // 1 Jan 2019
+static constexpr uint32_t kSecondsInADay        = 24 * 60 * 60;
+
+namespace {
+const uint8_t kMaxNtpServerStringSize = 128;
+char mSntpServerName[kMaxNtpServerStringSize];
```

Comment:
This is not a member. Please call it `sSntpServerName` since it's global (but `s`tatic to this module)

Reply:
Done